Week3/homework/update.js

```diff
@@ -0,0 +1,47 @@
+'use-strict';
+
+const validate = require('validate.js');
+const { isEmpty, isString } = require('validate.js');
+const connection = require('./connection');
+const { updateTodo, updateList, updateUser, Completed } = require('./que');
+const { allUsers, todoLists, todos } = require('./get');
+const { errorHandler, userError, listError, todoError } = require('./errorHandler');
+const constraints = {
+  from: {
+    email: true,
+  },
+};
+
+const user = async (req, res) => {
+  const body = req.body;
+  isEmpty(body) ||
```

Comment:
why is this isEmpty(body) useful?

Reply:
Hello @danlaudk ,
Thank you for your feed back .
isEmpty(body)  is not useful after i validated the user and the email but i did not notice that ...In the first i validated only the body and after few days i decide to validate the user and all required fields.... On that time i did not notice that validate the body will be not necessary any more  ....So it is my mistake...I should check my code carefully before submit my homework.
Best regards,